### PR TITLE
schema updates

### PR DESCRIPTION
**Pull request by moloko**
adds `_shouldRecordInteractions` into the schema to fix [#1494](https://github.com/adaptlearning/adapt_framework/issues/1494) 

also switch drop downs to checkboxes since it seems daft to make the user click twice to change the selection

amends to help text, spacing

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -1,70 +1,79 @@
...
-                  "type":"boolean",
-                  "required":true,
-                  "default": "true",
-                  "title":"Is Enabled",
-                  "inputType": {"type": "Boolean", "options": [true, false]},
+                  "type": "boolean",
+                  "required": true,
+                  "default": true,
+                  "title": "Is Enabled",
+                  "inputType": "Checkbox",
```
Comment (brian-learningpool) [inline properties.schema]: Replacing the select with a checkbox makes sense but I'm conscious that similar properties on other plugins still use <select> boxes and this introduces some inconsistency.

Comment (moloko) [inline properties.schema]: well, there is [a ticket to make them consistent](https://github.com/adaptlearning/adapt_authoring/issues/1380) that has been kicking around for ages so maybe this is a good opportunity to make that change?

Comment (moloko) [inline properties.schema]: also worth noting Helen's comment here https://github.com/adaptlearning/adapt_authoring/issues/1324

Comment (moloko) [inline properties.schema]: I would be happy to make this same change across all core schema plugins to make them consistent... just say the word!

Comment (oliverfoster) [inline properties.schema]: Kumquat!!?

Comment (moloko) [inline properties.schema]: @oliverfoster sorry that's not the word

I think you'll find the bird is the word... have you heard about the bird?

Comment (brian-learningpool) [inline properties.schema]: Hi @moloko, yes, I think this is a good opportunity to do that. I've updated the linked ticket just to keep track of it.

**Pull request merged**